src/app_service/common/cache.nim

```diff
@@ -14,7 +14,7 @@ proc newTimedCache*[T](): TimedCache[T] = initTable[string, Value[T]]()
 proc init*[T](self: var TimedCache[T], values: Table[string, T]) =
   self.clear()
   for cacheKey, value in values:
-    self[cacheKey].value = value
+    self.set(cacheKey, value)
```

Comment:
@igor-sirotin 

The original way assigned a value to the table without updating the timestamp, so whatever value we set will be considered "old" and `isCached` will return false immediately. If the user requires "fresh" values, they'll need to be fetched.
In the new way, calling the `set` method will update the timestamp, so the values will be considered "fresh" and `isCached` will return `true` for whatever duration is used, which is wrong since the values we used to init will probably be quite old.

I think the other changes should fix the issue and this one should not be necessary. I'll revert this change in a new PR and test to see that everything's still working fine.